Docs/user_guide/settings.md

```diff
@@ -69,6 +69,18 @@ List of modules that appear in the Favorites toolbar:
 
 To add a module, drag&drop it from the *Modules* list above. Then use the advanced panel (>>) to reorganize/delete the modules within the toolbar.
 
+### Appearance
+
+#### Style
+
+The overall theme of Slicer is controlled by the selected Style:  `Slicer`, `Light Slicer` or `Dark Slicer`
+
+`Slicer` is the default and it sets the style based on theme settings set by the operating system. For example, on Windows if [dark mode][windows-dark-mode] is turned on for apps, then the `Dark Slicer` style will be used upon launching Slicer.
```

Comment:
What do you think of being more explicit. Indeed after looking at the implementation of the Slicer style, it looks like the dark palette will only be used on windows when `Slicer` is the default style ? 

Raw:
```
The overall theme of Slicer is controlled by the selected Style:  `Slicer`, `Light Slicer` or `Dark Slicer`

`Slicer` is the default and it sets the style based on theme settings set by the operating system.

The table below summarizes how the appearance of the application is updated when the `Slicer` style is selected:

|             |  Dark Operating System Theme |  Light Operating System Theme |
|-------------|------------------------------|-------------------------------|
| **Linux**   | `Light Slicer`               | `Light Slicer`                |
| **macOS**   | `Light Slicer`               | `Light Slicer`                |
| **Windows** | `Dark Slicer`                | `Light Slicer`                |

`Light Slicer` and `Dark Slicer` are options used to override the system setting.


```

Rendered:

The overall theme of Slicer is controlled by the selected Style:  `Slicer`, `Light Slicer` or `Dark Slicer`

`Slicer` is the default and it sets the style based on theme settings set by the operating system.

The table below summarizes how the appearance of the application is updated when the `Slicer` style is selected:

|             |  Dark Operating System Theme |  Light Operating System Theme |
|-------------|------------------------------|-------------------------------|
| **Linux**   | `Light Slicer`               | `Light Slicer`                |
| **macOS**   | `Light Slicer`               | `Light Slicer`                |
| **Windows** | `Dark Slicer`                | `Light Slicer`                |

`Light Slicer` and `Dark Slicer` are options used to override the system setting.

Reply:
I wonder if there is a better way to convey the fact that `Light Slicer` will always be used on macOs and Linux when `Slicer` is the selected style.

Reply:
For users the behavior is the following: "Slicer theme" sets dark/light mode based on operating system settings. Slicer dark/light theme: dark/light mode is used, regardless of operating system settings. Current limitation: "Slicer theme" always uses light mode on Linux.

Reply:
Gotcha. I missed that comment:

https://github.com/Slicer/Slicer/blob/9bacb34fa5e26c02d79501af30d8c137ce2c4e4c/Base/QTGUI/qSlicerStyle.cxx#L199

How does this work ?

Reply:
After documenting this behavior, moving forward with the integration of this makes sense.

Reply:
That said, should we should keep `Slicer Light` ? 

If not, should we introduce a build settings to change the default ? In context of custom  application, it is important to ensure a style is consistently used.

Reply:
Yes, we need all three themes:
- Slicer: automatic bright or dark (based on OS settings)
- Slicer Light: always light (regardless of OS settings)
- Slicer Dark: always dark (regardless of OS settings)

Reply:
Agreed, I realized my last comment was confusing. 

Instead of `should we should keep Slicer Light ?`, I meant to ask if  we should keep `Slicer Light` as the default theme ? 

If not, I suggest we introduce a build setting allowing to  choose which theme is the default one.